main.go

```diff
@@ -5,7 +5,7 @@ import (
 	"os"
 	"time"
 
-	"github.com/urfave/cli"
+	"github.com/codegangsta/cli"
```

Comment:
Let's continue using urfave/cli. That's the canonical name for that CLI package.

```suggestion
	"github.com/urfave/cli"
```